Readme updated (#6)
molgenis/AGCT: README.md

#  Array Genotyping Conversion Tool

The AGCT (Array Genotyping Conversion Tool) pipeline consists of only 1 step:
1: Convert IDAT to GTC Format
```
This step takes IDAT files (Illumina SNP array scan Files) and converts them to GTC Format.

> For this to happen an reference file (in .egt format) to normalise the data is needed and also the manifest file (in .bpm format) are needed.

Reviewer: ```suggestion
For this to happen, a reference file (in .egt format) to normalize the data is needed and also the manifest file (in .bpm format) is needed.
```

> The bpm file consists of information which probes lie on the array platform.

Reviewer: ```suggestion
The bpm file contains information about which probes lie on the array platform.
```
